src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -325,7 +322,7 @@ export default {
     focusSearch() {
       this.$nextTick(() => {
         /* istanbul ignore else */
-        if (this.isOpen && this.$refs.searchInput) {
```

Comment:
if `isSearchEnabled` then `this.$refs.searchInput` always exists, we should use the former for conditional logic